src/result_processor.h

```diff
@@ -75,6 +75,10 @@ typedef struct {
   // Contains our spec
   RedisSearchCtx *sctx;
 
+  bool isProfile;
```

Comment:
consider moving this member next to `timeoutPolicy`, so we have less padding in the struct